example/convolution1D/src/convolution1D.cpp

```diff
@@ -0,0 +1,178 @@
+/* Copyright 2023 Benjamin Worpitz, Erik Zenker, Bernhard Manfred Gruber, Jan Stephan
```

Comment:
You may want to add yourself here. Also, I am not sure if everybody on this list contributed to your example :)

Reply:
What is the policy about this? So I am adding only current team members like 
_"Copyright 2023  Bernhard Manfred Gruber, Simeon Ehrig, Rene Widera, Mehmet Yusufoglu."_

Reply:
I cannot give you legal advice. In principle, if you create something original, then you hold copyright for this creation. I assume you created at least a part of the 1D Convolution here, so you hold copyright to this "invention". If you copied some pieces of code from somewhere else, other people may hold copyright too, which you have to include. If you just included small changes from elsewhere, like a few lines of code that cannot be called an "original work", you do not need to name the author. Here are some [guidelines from the GPU project](https://www.gnu.org/prep/maintain/html_node/Legally-Significant.html).

Reply:
However, you may find some people on the team, that are pedantic about adding anybody to the copyright header who flipped a single character in the source file :) It's ultimately also some amount of personal judgement.

But for this case here: if you didn't fully copy your example from the web, I am sure it contains original work from you!